lib/external/npm.ts

```diff
@@ -0,0 +1,39 @@
+import {get} from "request-promise-native";
+
+export class NpmClient {
+
+    public static async getLastVersion(): Promise<string> {
+        const options = {
+            uri: "https://registry.npmjs.org/bespoken-tools/latest",
+            timeout: 30000
```

Comment:
we should wait less, 3000 should be enough (we don't want to the user to have delays if npm is unreacheable)